### PR TITLE
Added new ebuild for dev-python/matrix2latex:

### PR DESCRIPTION
**Pull request by TheChymera**
A small package for converting Python or MATLAB matrices to LaTeX booktabs. Very useful in conjunction with PythonTeX.

### CODE REVIEW

File: dev-python/matrix2latex/matrix2latex-0.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python{2_7,3_2,3_3} )
+
+inherit python-single-r1
+
+DESCRIPTION="A tool to create LaTeX tables from python lists and arrays."
```
Comment (jlec) [inline dev-python/matrix2latex/matrix2latex-0.1.ebuild]: Drop ```.```from here

File: dev-python/matrix2latex/matrix2latex-0.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python{2_7,3_2,3_3} )
```
Comment (jlec) [inline dev-python/matrix2latex/matrix2latex-0.1.ebuild]: pyhton3_2 support is gone, but python3_4 is there.

File: dev-python/matrix2latex/matrix2latex-0.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+DESCRIPTION="A tool to create LaTeX tables from python lists and arrays."
+HOMEPAGE="https://code.google.com/p/matrix2latex/"
+SRC_URI="https://matrix2latex.googlecode.com/files/"${PN}"Python2011-12-15.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
```
Comment (jlec) [inline dev-python/matrix2latex/matrix2latex-0.1.ebuild]: Missing ```REQUIRED_USE```

File: dev-python/matrix2latex/matrix2latex-0.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+RDEPEND="app-text/texlive"
+DEPEND=""
```
Comment (jlec) [inline dev-python/matrix2latex/matrix2latex-0.1.ebuild]: Missing python deps.

File: dev-python/matrix2latex/matrix2latex-0.1.ebuild [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+src_unpack() {
+	if [ ${A} != "" ]; then
+		unpack ${A}
+	fi
+	S="${WORKDIR}/${PN}Python"
+	cd "${S}"
+}
```
Comment (jlec) [inline dev-python/matrix2latex/matrix2latex-0.1.ebuild]: What is this function good for?

File: dev-python/matrix2latex/matrix2latex-0.1.ebuild [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+EAPI=5
+
+PYTHON_COMPAT=( python{2_7,3_2,3_3} )
+
+inherit python-single-r1
+
+DESCRIPTION="A tool to create LaTeX tables from python lists and arrays."
+HOMEPAGE="https://code.google.com/p/matrix2latex/"
+SRC_URI="https://matrix2latex.googlecode.com/files/"${PN}"Python2011-12-15.tar.gz"
```
Comment (jlec) [inline dev-python/matrix2latex/matrix2latex-0.1.ebuild]: Why are you putting quotes around ${PN}?

File: dev-python/matrix2latex/matrix2latex-0.1.ebuild [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python{2_7,3_2,3_3} )
+
+inherit python-single-r1
+
+DESCRIPTION="A tool to create LaTeX tables from python lists and arrays."
+HOMEPAGE="https://code.google.com/p/matrix2latex/"
```
Comment (jlec) [inline dev-python/matrix2latex/matrix2latex-0.1.ebuild]: google code is dead.

File: dev-python/matrix2latex/matrix2latex-0.1.ebuild [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+	S="${WORKDIR}/${PN}Python"
+	cd "${S}"
+}
+
+src_install() {
+	python_optimize .
+	python_domodule "${S}"/matrix2latex.py
+	python_domodule "${S}"/fixEngineeringNotation.py
+	python_domodule "${S}"/error.py
+	python_domodule "${S}"/IOString.py
```
Comment (jlec) [inline dev-python/matrix2latex/matrix2latex-0.1.ebuild]: This looks heavily like a bundled IOString module.

File: dev-python/matrix2latex/matrix2latex-0.1.ebuild [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+	fi
+	S="${WORKDIR}/${PN}Python"
+	cd "${S}"
+}
+
+src_install() {
+	python_optimize .
+	python_domodule "${S}"/matrix2latex.py
+	python_domodule "${S}"/fixEngineeringNotation.py
+	python_domodule "${S}"/error.py
```
Comment (jlec) [inline dev-python/matrix2latex/matrix2latex-0.1.ebuild]: You could put them all in one call.

File: dev-python/matrix2latex/matrix2latex-9999.ebuild [hunk 10]
```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python{2_7,3_2,3_3} )
```
Comment (jlec) [inline dev-python/matrix2latex/matrix2latex-9999.ebuild]: See above.

File: dev-python/matrix2latex/matrix2latex-9999.ebuild [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+
+inherit distutils-r1 git-r3 multilib
+
+DESCRIPTION="A tool to create LaTeX tables from python lists and arrays."
```
Comment (jlec) [inline dev-python/matrix2latex/matrix2latex-9999.ebuild]: Here as well

File: dev-python/matrix2latex/matrix2latex-0.1.ebuild (hunk 5 above)
Comment (TheChymera) [inline dev-python/matrix2latex/matrix2latex-0.1.ebuild]: unpacks and specifies the source for the package (if I remove it, portage will fail due to not finding ${S}.

File: dev-python/matrix2latex/matrix2latex-0.1.ebuild (hunk 8 above)
Comment (TheChymera) [inline dev-python/matrix2latex/matrix2latex-0.1.ebuild]: if I remove it from the list `import matrix2latex` will fail:

```
>>> import matrix2latex
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python2.7/site-packages/matrix2latex.py", line 22, in <module>
    from IOString import IOString
ImportError: No module named IOString
```

**Pull request closed without merging**